protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3305,3 +3314,66 @@ message PingRequest {
 // Ping response from the data node
 message PingResponse {
 }
+
+// Basic description of an order.
+message OrderInfo { 
+ // Side for the order, e.g. buy or sell.
+  vega.Side side = 1;
+  // Price for the order. The price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  // This field is an unsigned integer passed as a string and needs to be scaled using the market's decimal places.
+  string price = 2;
+  // Size remaining, when this reaches 0 then the order is fully filled and status becomes filled.
+  // This field is an unsigned integer passed as a string and needs to be scaled using the market's position decimal places.
+  uint64 remaining = 3;
+  // Boolean that indicates if it is a market order.
+  bool is_market_order = 4;
+}
+
+
+// Request for the estimate of the margin level and, if available collateral is provided, liqudation price for the specified position.
+message EstimatePositionRequest {
+     // Market identifier.
```

Comment:
```suggestion
     // Market ID.
```

Reply:
This field is labeled as "Market identifier" in most cases in .proto.

Reply:
The API clean ups in: https://github.com/vegaprotocol/vega/pull/8126

Try to make this consistent across all comments that become API docs.

Reply:
Ah ok, didn't know - thanks! Committed your suggestions now.